EventHubs/src/azuredeploy_metrics_with_appinsights.json

```diff
@@ -1,5 +1,5 @@
 {
-    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
```

Comment:
lets delete this file and copy paste it's code to azuredeploy_metrics.json file we will have single file.